ldaptor/encoder.py

```diff
@@ -11,3 +13,23 @@ def to_bytes(value):
     if isinstance(value, six.text_type):
         return value.encode('utf-8')
     return bytes(value)
+
+
+def to_unicode(value):
+    if isinstance(value, six.binary_type):
+        return value.decode('utf-8')
+    return value
+
+
+class WireStrAlias(object):
+    def __str__(self):
+        warnings.simplefilter('always', DeprecationWarning)
+        warnings.warn('{0}.__str__ method is deprecated and will not be used '
+                      'for getting bytes representation in the future '
+                      'releases, use {0}.toWire instead'.format(self.__class__.__name__),
+                      category=DeprecationWarning)
+        warnings.simplefilter('default', DeprecationWarning)
+        return self.toWire()
+
+    def toWire(self):
+        raise NotImplementedError()
```

Comment:
It would be nice to add an helper text to this exception.
That is not instantiate an empty NotImplementedError but instead provide a clue.
For me, it makes it easier to detect what is wrong, without having to search for the traceback.

```suggestion
        raise NotImplementedError('toWire not implemented')
```

Reply:
Got it, fixed!